std_msgs/Int32 redPoints
std_msgs/Int32 bluePoints
ChangeUpGoals goals
std_msgs/Int8[] goalsColor
